src/main/scala/io/appthreat/atom/slicing/DataFlowSlicing.scala

```diff
@@ -20,7 +20,7 @@ object DataFlowSlicing {
     (config.fileFilter match {
       case Some(fileName) => cpg.file.nameExact(fileName).ast.isCall
       case None           => cpg.call
-    }).iterator.method.withMethodNameFilter.withMethodParameterFilter.withMethodAnnotationFilter.ast.isCall
+    }).where(c => c.callee.isExternal)
```

Comment:
These methods you are removing are rendering a bunch of filters useless

```suggestion
}).iterator.method.withMethodNameFilter.withMethodParameterFilter.withMethodAnnotationFilter.ast.isCall.where(c => c.callee.isExternal)
```